src/Sweep.cpp

```diff
@@ -1404,46 +1327,40 @@ int TreatSweep(double t)
 {
 	///// function loops over microcells to decide which cells are treated (either with treatment, vaccine, social distancing, movement restrictions etc.)
 
-	int TreatFlag, TreatFlag1; //// Function returns TreatFlag. If TreatFlag == 0, function no longer called. Anytime any treatment used, TreatFlag set to 1. 
-	int f2, f3, f4; //// various fail conditions. Used for other things
+	int TreatFlag = 0, TreatFlag1 = 0; //// Function returns TreatFlag. If TreatFlag == 0, function no longer called. Anytime any treatment used, TreatFlag set to 1. 
 	int nckwp;
 
 	//// time steps
-	unsigned short int ts;								////  time-step now.
-	unsigned short int t_TreatStart;					////  time-step treatment begin
-	unsigned short int t_TreatEnd;						////  time-step treatment finish
-	unsigned short int t_VacStart;						////  time-step vaccination begin
-	unsigned short int t_PlaceClosure_End;				////  time-step place closure finish
-	unsigned short int t_MoveRestrict_Start;			////  time-step movement restriction begin
-	unsigned short int t_MoveRestrict_End;				////  time-step movement restriction finish
-	unsigned short int t_SocDist_End;					////  time-step social distancing finish
-	unsigned short int t_KeyWorkerPlaceClosure_End;		////  time-step key worker place closure finish
-	int global_trig;
-	double r;
+	unsigned short int TimeStepNow = (unsigned short int) (P.TimeStepsPerDay * t);	////  time-step now.
+	unsigned short int t_TreatStart;												////  time-step treatment begin
+	unsigned short int t_TreatEnd;													////  time-step treatment finish
+	unsigned short int t_VacStart;													////  time-step vaccination begin
+	unsigned short int t_PlaceClosure_End;											////  time-step place closure finish
+	unsigned short int t_MoveRestrict_Start;										////  time-step movement restriction begin
+	unsigned short int t_MoveRestrict_End;											////  time-step movement restriction finish
+	unsigned short int t_SocDist_End;												////  time-step social distancing finish
+	unsigned short int t_KeyWorkerPlaceClosure_End;									////  time-step key worker place closure finish
+	double radius;
 
-	ts = (unsigned short int) (P.TimeStepsPerDay * t);
-	TreatFlag = TreatFlag1 = 0;
+	int global_trig;
 	if (P.DoGlobalTriggers)
 	{
-		if (P.DoPerCapitaTriggers)
-			global_trig = (int)floor(((double)State.trigDetectedCases) * P.GlobalIncThreshPop / ((double)P.PopSize));
-		else
-			global_trig = State.trigDetectedCases;
+		if (P.DoPerCapitaTriggers)	global_trig = (int)floor(((double)State.trigDetectedCases) * P.GlobalIncThreshPop / ((double)P.PopSize));
+		else						global_trig = State.trigDetectedCases;
 	}
-	else
-		global_trig = 0;
+	else							global_trig = 0;
```

Comment:
int global_trig = P.DoGlobalTrigger ? (P.DoPerCapitaTriggers ? (int)floor ...  : State.trigDetectedCases) : 0;

if you liike,,,

Or less extreme, int global_trig = 0; on 1345, and delete 1351